tiledb/sm/misc/constants.cc

```diff
@@ -723,6 +723,15 @@ const std::string filestore_metadata_original_filename_key =
 /** Name of the metadata key used in filestore arrays for filename extension. */
 const std::string filestore_metadata_file_extension_key = "file_extension";
 
+/** The minimum size of an s3 multipart upload intermediate part */
+const uint64_t s3_min_multipart_part_size = 5242880;
+
+/**
+ * The dir name under fragment uri used by s3 to store remote
+ * global order writes intermediate chunks
+ */
+const std::string s3_multipart_buffering_dirname = "__gow_chunks";
```

Comment:
Is there any issues spelling gow fully? If anyone that doesn't know our internals too well sees this folder, it might help give more clues to what it is.